packages/utils/src/formatters/formatDataForMutation.js

```diff
@@ -0,0 +1,136 @@
+
+          formatedFieldData = formatedFieldData.map((data) => formatDataForCreateMutation(relationTableSchema.name, data, schema));
+
+          formatedFieldData = { create: formatedFieldData };
+        }
+      } else {
+        if (typeof data === 'string') {
+          formatedFieldData = { connect: formatedFieldData };
+        } else {
+          const relationTableSchema = selectors.getTableSchemaById(fieldSchema.relation.refTable.id, schema);
+
+          formatedFieldData = formatDataForCreateMutation(relationTableSchema.name, formatedFieldData, schema);
+
+          formatedFieldData = { create: formatedFieldData };
+        }
+      }
+    }
+  } else {
+    throw new Error(`Field schema with ${fieldName} name not found in table schema with ${tableSchema.name} name.`);
+  }
+
+  return formatedFieldData;
+};
+
+const formatDataForCreateMutation = (tableName: string, data: Object, schema: Schema) => {
+  const tableSchema: ?TableSchema = selectors.getTableSchemaByName(tableName, schema);
+
+  let formatedData = null;
+
+  if (tableSchema) {
+    formatedData = R.mapObjIndexed(
+      formatFieldDataForCreateMutation(tableSchema, schema),
+      data,
+    );
+  } else {
+    throw new Error(`Table schema with ${tableName} name not found in schema.`);
+  }
+
+  return formatedData;
+};
+
+const formatFieldDataForUpdateMutation = (tableSchema: TableSchema, schema: Schema) => (data: Object, fieldName: string) => {
+  const fieldSchema: ?FieldSchema = selectors.getFieldSchemaByName(fieldName, tableSchema);
+
+  let formatedFieldData = data;
+
+  if (fieldSchema) {
+    if (verifiers.isRelationField(fieldSchema)) {
+      if (verifiers.isListField(fieldSchema)) {
+        if (data.length === 0 || typeof data[0] === 'string') {
+          formatedFieldData = { reconnect: formatedFieldData };
+        } else {
+          const relationTableSchema = selectors.getTableSchemaById(fieldSchema.relation.refTable.id, schema);
+
+          formatedFieldData = formatedFieldData.map((data) => formatDataForUpdateMutation(relationTableSchema.name, data, schema));
+
+          formatedFieldData = { create: formatedFieldData };
+        }
+      } else {
+        if (typeof data === 'string') {
+          formatedFieldData = { reconnect: formatedFieldData };
+        } else {
+          const relationTableSchema = selectors.getTableSchemaById(fieldSchema.relation.refTable.id, schema);
+
+          formatedFieldData = formatDataForUpdateMutation(relationTableSchema.name, formatedFieldData, schema);
+
+          formatedFieldData = { create: formatedFieldData };
+        }
+      }
+    }
+  } else {
+    throw new Error(`Field schema with ${fieldName} name not found in table schema with ${tableSchema.name} name.`);
+  }
+
+  return formatedFieldData;
+};
+
+const formatDataForUpdateMutation = (tableName: string, data: Object, schema: Schema) => {
```

Comment:
This duplicates the logic of the `formatDataForCreateMutation ` function. Maybe, refactor?